pipe.py

```diff
@@ -71,6 +71,7 @@
     "transpose",
     "dedup",
     "uniq",
+    "filter"
```

Comment:
```suggestion
    "filter",
```

just so the next diff won't look like:

```diff
-    "filter"
+    "filter",
+    "the new thing"
```

but simply:

```diff
+    "the new thing"
```